demo/admin/src/common/MasterMenu.tsx

```diff
@@ -165,9 +165,9 @@ export const masterMenuData: MasterMenuData = [
         requiredPermission: "pageTree",
     },
     {
-        primary: <FormattedMessage id="menu.products" defaultMessage="Products" />,
+        title: <FormattedMessage id="menu.products" defaultMessage="Products" />,
         icon: <Snips />,
-        submenu: [
+        groupItems: [
```

Comment:
I'm not sure if i like this differentiation based on fields. Wouldn't an explicit `type: "group" | "route" | "anchor" be better?

Reply:
Thought the same but i stuck to the original implementation. Should I change it?

Reply:
https://github.com/vivid-planet/comet/pull/2080/commits/79e8b916661d7b566dbbfa8480543a98be5038f8

Reply:
Now that we differentiate based on type, I'd name this `items`

Reply:
Renamed in 0b789f0d02101a66d88aba6b66c3241221672070